README.md

```diff
@@ -58,7 +56,7 @@ Sync project with gradle files. (Android Studio -> File -> Sync Project With Gra
 
 ### Quick Sample 
 
-Create `CallComposite` and launch it. Replace `<GROUP_CALL_ID>` with your group ID for your call, `<DISPLAY_NAME>` with your name, and  `<USER_ACCESS_TOKEN>` with your token.
+Create `CallComposite` and launch it. Replace `<GROUP_CALL_ID>` with your group ID for your call, `<DISPLAY_NAME>` with your name, and  `<USER_ACCESS_TOKEN>` with your token. For full instructions check out our [quickstart](https://docs.microsoft.com/azure/communication-services/quickstarts/ui-library/get-started-composites?tabs=kotlin&pivots=platform-android) or get the completed [sample](https://github.com/Azure-Samples/communication-services-android-quickstarts/tree/main/ui-library-quick-start).
```

Comment:
Should add this link to a link table at the bottom in the "Further Reading" as well.

Reply:
Thanks for pointing it out Adam. Adding it.